cmd/zot/main_test.go

```diff
@@ -4,14 +4,15 @@ import (
 	"testing"
 
 	"github.com/anuvu/zot/pkg/api"
+	"github.com/anuvu/zot/pkg/api/config"
 	"github.com/anuvu/zot/pkg/cli"
 	. "github.com/smartystreets/goconvey/convey"
 )
 
 func TestIntegration(t *testing.T) {
 	Convey("Make a new controller", t, func() {
-		config := api.NewConfig()
-		c := api.NewController(config)
+		conf := config.New()
```

Comment:
why this refactor?

Reply:
I needed to refactor config because I needed certs inside sync package, and because config was part of api package which imports extensions => circular import.